source/cloud/cloud-administration/cloud-changelog.md

```diff
@@ -0,0 +1,86 @@
+# Mattermost Cloud Changelog
+
+This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.
+
+## Release 2020-12-09
+
+### Improvements
+
+#### User Interface (UI)
+ - Added the ability to mute categories with the experimental sidebar feature.
+ - Added support for multi-selection of channels for dragging and dropping between channels in the experimental sidebar feature.
+ - Group messages are now returned in the channel switcher when only first names are typed.
+ - /dnd command will only set a do not disturb status whatever the current user status is.
+
+#### Administration
+ - Added a new `manage_remote_clusters` permission.
+ - Enabled goSAML2 library as the only supported SAML Library.
+
+### Bug Fixes
+ - Cleaned up the config store on server initialization errors.
+ - Fixed an issue where permissions did not grant read and/or write access to the Global Relay configuration settings.
+ - Fixed an issue where the site configuration ‘’Read only’’ permission did not make the "Notice" section as read-only for the System Manager.
+ - Fixed an issue where importing Client4 in a node server caused an exception due to rudder modules.
+ - Fixed an issue where LDAP ‘’FirstLoginSync’’ didn't close LDAP Session.
+
+### Known Issues
+ - System Manager does not have access to Billing section and sees blank screen.
+ - Cloud > "Tips & Next Steps" should not show "Explore channels" section for guest users.
+ - System Roles shows License and Environment as possible permissions but are always hidden in Cloud
+
+## Release 2020-12-03
+
+### Bug Fixes
+  - Disabled the xmlsec1-based SAML library in favor of the re-enabled and improved SAML library.
+
+## Release 2020-11-24
+
+### Highlights
+
+#### Modify new Admin roles permissions in the System Console (Beta)
+ - Added a user interface for the existing new system roles functionality including a new permission to read and write system_roles.
+
+#### Pre-package and enable incident management v1.1.1
+
+### Improvements
+
+#### User Interface
+ - @-autocomplete results are now prioritized based on recency and thread activity.
+ - File attachments below size of 10 now show fractions.
```

Comment:
This bullet point is unclear. 
- Does "below size of 10" refer to the number of file attachments or the size of the file attachments?
- the term "fractions" suggest that 1/2, 1/3, 1/4 is shown; is that the intention?

Reply:
I adjusted the wording to "File attachments below the size of 10 (KB, MB, GB, TB, etc.) now allow showing fractions." @svelle Anything else to add?

Reply:
Ticket here with more details https://mattermost.atlassian.net/browse/MM-30193.